src/base/io/convertModelToEX.m

```diff
@@ -51,6 +50,7 @@ function convertModelToEX(model, filename, rxnzero, EXrxns)
         reactionPlace = find(model.S(:, i));
         if abs(model.S(reactionPlace, i)) > 1 - 1e-2
             for j = 1:size(reactionPlace, 1)
+              model.S=full(model.S);
```

Comment:
Is this absolutely necessary? For large models, this might cause out-of-memory issues. Also, why is this done in the loop at each iteration?

Reply:
The operation shouldn't be there, true. I will move it to a better place, but without it, the 'fprintf' function cannot work - it won't operate on sparse arguments.

Reply:
Not sure, whether you already plan this: But you could simply call fprintf(...,full(model.S(x,y)),...)
This way you only convert one single entry each time instead of the full matrix.

Reply:
I changed the function in two places as you suggested